build-deb-package

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file BOOST_LICENSE_1.0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -xeu
+
+# if the requested build identifier matches a git tag, use that directly,
+# otherwise use git revision description (note, we implicitly assume that all
+# tags and revision descriptions will begin with `v`)
+versiontag=$(git describe)
+
+# Set up package
+pkgname=libmxnet
+pkgversion_number=$(echo "${versiontag}" | sed 's/^v\(.*\)$/\1/' ) # remove any initial `v`
+pkgdistro=$(lsb_release -cs)
+pkgversion="${pkgversion_number}-${pkgdistro}"
+pkgmaintainer="Sociomantic Tsunami <tsunami@sociomantic.com>"
+
+DEB_HOST_MULTIARCH=x86_64-linux-gnu
```

Comment:
Yea, probably a leftover of some copy-paste from elsewhere.  I'll cut, and restore if needed.